mitmproxy/contentviews/json.py

```diff
@@ -21,7 +22,33 @@ class ViewJSON(base.View):
         "application/vnd.api+json"
     ]
 
+    @staticmethod
+    def _format(data):
+        lines = data.splitlines()
+        for line in lines:
+            li = []
+            decode_byte = line.decode("utf8")
```

Comment:
`return pretty.encode("utf8", "strict")
`
As pretty JSON returns utf8 encoded string, that's why it is decoded with utf8 format but if we are not following pretty JSON then it can be removed.In hypothesis library there is no direct strategies for JSON so we can either use an extended library like [hypothesis-json](https://pypi.org/project/hypothesis-json/) or use recursive approach as given in [hypothesis documentation](https://hypothesis.readthedocs.io/en/1.10.0/data.html#recursive-data) but for both of them we have to encode the data because ViewJSON accept data as bytes.What do you think which approach should we follow?

Reply:
Yeah, we're probably safe here due to the checks in pretty_json. My idea was that we could use hypothesis to check that we don't crash on random bytes as input. No need to use any fancy JSON-generation. :)